civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php

```diff
@@ -6,25 +6,27 @@
 </div>
 <?php endif; ?>
 
-<ul class="nav nav-tabs nav-justified report-tabs">
+<ul class="nav nav-tabs nav-justified report-tabs panel chr_tabs__header">
```

Comment:
Also, why mixing a `panel` with `nav-tabs`? Is it standard practice in Bootstrap?